### ISSUE TITLE
Community: Provide suggestions for content/structure of "News & Events" page

### ISSUE THREAD

**Issue by Erin-Cecele** [gettypubs/quire#241]
IDEA: split into two pages. 

Content of NEWS page should reflect newsletter with entries running chronologically almost like a blog, similar to Arches: https://www.archesproject.org/news/. This prevents duplicating efforts. 

My main concern with the separate EVENTS page is what to do when there are no upcoming events. Can we determine some placeholder content? Or we hold monthly webinars/workshops?

**Comment by geealbers** [comment]
I think keeping them on the same page should be ok. At least for now. I made an update with a possible approach. 

https://deploy-preview-301--quire.netlify.app/community/news-events/

Here I'm treating the News section more as a list of milestones, with links when applicable, and then providing a general link to the newsletter for people to get more detail. I think you're right that we don't want to duplicate efforts/content and I'd expect mostly people to be getting news from the email, not checking in on this webpage regularly. 

And for events, I used a table format, which I think works ok. The other option would be to move it all to YAML and create a new shortcode, but that feels like a little more than is maybe necessary.
